kimnyamhondera/blink182/blink182.ino

```diff
@@ -0,0 +1,18 @@
+/* This sketch is sketch for sketch this sketch task sketch https://trello.com/c/ba783WmU/2-blink-an-led */
```

Comment:
I forgot to change this, I did this before i knew we'd be sharing it with the whole group